packages/cli/src/commands/index.ts

```diff
@@ -40,6 +45,11 @@ export function registerCommands(program: CommanderStatic) {
     .allowUnknownOption(true)
     .helpOption(', --backstage-cli-help') // Let docker handle --help
     .option('--build', 'Build packages before packing them into the image')
+    .option(
+      '--build-parallel [value]',
```

Comment:
Let's not add this 😅 . This is pretty easy to override by omitting the --build flag and calling lerna manually. It's really more of a local development utility rather than something you should rely on in CI. It's was added to avoid the confusion of stale builds in docker images that we where getting quite a lot of reports for.